packages/core-components/src/components/Link/Link.test.tsx

```diff
@@ -134,7 +134,7 @@ describe('<Link />', () => {
   });
 
   describe('useResolvedPath', () => {
-    const wrapper: WrapperComponent<{}> = ({ children }) => {
+    const wrapper: React.FC<React.PropsWithChildren<{}>> = ({ children }) => {
```

Comment:
Heads-up for the following [`ADR` on `React.FC`](https://backstage.io/docs/architecture-decisions/adrs-adr006)